src/components/Navigation.tsx

```diff
@@ -66,6 +66,9 @@ const Navigation: React.FC<Props> = props => {
                   ))}
               </ul>
             </nav>
+            <a href="https://www.facebook.com/hartaserviciilorsociale" title="Facebook Page">
```

Comment:
pls fa un search dupa linkuri externe in site si adauga `target="_blank" rel="noreferrer noopener"` la ele.